modules/knative-serving-CR-system-deployments.adoc

```diff
@@ -33,6 +34,14 @@ spec:
     replicas: 2
   deployments:
   - name: webhook
+    resources:
+    - container: activator
```

Comment:
The deployment is called "webhook" and it only has a container called "webhook". Let's change this line to `- container: webhook`.
The rest looks ok to me.